src/microbe_stage/editor/EditorComponentBottomLeftButtons.cs

```diff
@@ -267,6 +276,11 @@ private void PerformValidation(string text)
         }
     }
 
+    private bool ValidateNameSize(string name)
+    {
+        return speciesNameEdit.GetFont("font").GetStringSize(name).x < speciesNameEdit.RectSize.x;
```

Comment:
Specifically checking the size against the line edit's length I don't think is the best. Could you instead put a new float constant in `Constants` that this is checked against, then it can be easily tweaked if we find places elsewhere in the GUI long names don't really work.

Reply:
how long roughly should that constant be? I'll set it to 123 pixels just as a start.

Reply:
I set it to 230, which is slightly less than the default Rect.x size of the name input text box (240).